Sources/Datadog/RUM/DataModels/RUMErrorEvent.swift

```diff
@@ -0,0 +1,68 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+// TODO: RUMM-517 Replace with auto-generated RUM data model
+internal struct RUMErrorEvent: Codable, RUMDataModel {
+    let date: UInt64
+    let application: Application
+    let session: Session
+    let type = "error"
+    let view: View
+    let error: Error
+    let action: Action?
+    let dd: DD
+
+    struct Application: Codable {
+        let id: String
+    }
+
+    struct Session: Codable {
+        let id: String
+        let type: String
+    }
+
+    struct View: Codable {
```

Comment:
Yes, this will change when we switch to the generated code. But AFAIK `View` values are slightly different for various RUM events so it's possible that they will be declared as separate types following the schema.

Reply:
there are [`RUMActionView`](https://github.com/DataDog/dd-sdk-ios/blob/rum/Sources/Datadog/RUM/DataModels/RUMDataModels.swift#L273) and [`RUMViewView`](https://github.com/DataDog/dd-sdk-ios/blob/rum/Sources/Datadog/RUM/DataModels/RUMDataModels.swift#L723)
only `view-schema.json` has a different `view` property, the others use the same `view` (`RUMActionView`)